### ISSUE TITLE
Homepage cannot be opened owing to plugin unavailability

### ISSUE THREAD

**Issue by PSG2017** [mirnovov/obsidian-homepage#111]
### Details

Recently when I click on the Home icon (the house symbol) in the Obsidian sidebar I get the above error message, and the page does not open. Within the last week the icon was working. I have not added any new plugins since then. When I find the page designated as the homepage in the list of notes, and click it, it opens as normal. Does anyone have any ideas what could be causing this issue? Obsidian and all plugins are up to date.

### Operating systems

Windows

### Checklist

- [X] I updated to the latest version of the plugin.

**Comment by PSG2017** [comment]
I have now been able to fix this. The designation of the homepage file in the settings had been changed to Daily Note, so I had to change it back to File and specify the particular file (again). I don't know but perhaps this change happened during an update of the plugin (as I had not changed anything manually)?

**Comment by mirnovov** [comment]
Yeah, that message appears when it’s set to a plug-in that isn’t installed. No idea why it would have changed to that when you didn’t set it that way though, perhaps this was caused by another plugin. There is logic to upgrade so older types of homepage to Daily Notes when upgrading, so there may also be an edge case where the logic was accidentally triggered.
